newrelic.yml

```diff
@@ -245,9 +255,9 @@ common: &default_settings
   # If true, the agent captures custom events.
   # custom_insights_events.enabled: true
 
-  # * Specify a maximum number of custom events to buffer in memory at a time.
-  # * When configuring the agent for AI monitoring, set to max value 100000. This
-  # ensures the agent captures the maximum amount of LLM events.
+  # - Specify a maximum number of custom events to buffer in memory at a time.
+  #   - When configuring the agent for AI monitoring, set to max value 100000.
```

Comment:
Is this one supposed to be indented?

Reply:
Theres a comment a lil above this:
`# Keep the extra two-space indent before the second bullet to appease translation tool`

I don't know why the translation tool doesn't like this case, but decided to leave it alone.

Reply:
Whoops, missed that! Thank you!

Reply:
I almost missed it too!